pmsoService/product_flow/features/deliverytrip/tests/unit/test_handlers.py

```
@@ -0,0 +1,471 @@
+        self.sale_order.status = OrderStatus.COMPLETED.value
+        self.sale_order.save()
+        item_command = self.helpers.create_delivery_item_command(
+            sale_order_item_id=str(self.sale_order_item.id),
+            item_kind="PRODUCT",
+            quantity=5,
+        )
+        command = self.helpers.create_bulk_add_command(
+            delivery_trip_id=str(self.delivery_trip.id),
+            items=[item_command],
+        )
+        with self.assertRaises(Exception):  # ValidationError
```

Ruff (0.14.4)

Do not assert blind exception: `Exception`

(B017)